.changelog/4365.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Modified the PB125 validation to follow the new validation format.
```

Comment:
```suggestion
- description: Modified the PB125 validation to follow the new validation format. The validation <what it does>.
```

Reply:
@samuelFain Are we ok with this?
I saw you [removed](https://github.com/demisto/demisto-sdk/pull/4344/files) such comments from changelog.

Reply:
This was a mistake, my bad.
You can return the validation description.